example/client/include/IMessageConsumer.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "bftclient/base_types.h"
+
+namespace concord::kvbc {
+
+class IMessageConsumer {
+ public:
+  virtual ~IMessageConsumer() = default;
+  virtual void executeMessage() = 0;
+  virtual void deserializeWriteRequestMessage(const bft::client::Reply& rep) = 0;
```

Comment:
Is it request or response.
can we have a single function called deserialize(rep, bool readonly)

Reply:
fixed